OmsAgent/omsagent.py

```diff
@@ -1043,25 +1043,30 @@ def run_command_and_log(cmd, check_error = True, log_cmd = True):
         if exit_code is not 0:	
             sys.stderr.write(output[-500:])        
 
-        if exit_code is 19:
-            if "rpmdb" in output or "libc6 is not installed" in output or "libpam-runtime is not installed" in output or "cannot open Packages database" in output or "exited with status 52" in output or "/bin/sh is needed" in output:
+        if exit_code is 17:
+            if "Failed dependencies:" in output or "waiting for transaction lock" in output or "dpkg: error processing package systemd" in output:
```

Comment:
for this case is there any useful logging to emit to hint to cx/ux that something is corrupt or needs to be installed?

Reply:
STDOUT and the exit code will be shown on the portal and will be useful in debugging the failure